### PR TITLE
UIIN-1714 JEST/RTL test for Confirmation.js file

### PR DESCRIPTION
**Pull request by KetineniM**
Refs:
Ticket: [UIIN-1714]
URL: https://issues.folio.org/browse/UIIN-1714

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

**Review by ssandupatla — APPROVED** [review]

**Review by VSnehalatha — APPROVED** [review]

**Comment by zburke** [comment]
Admittedly, the PR template isn't a great fit for PRs like this where you're just adding tests. If you choose to leave it in place, however, you need to fill in all the questions. Alternatively, remove it and provide your own description of the changes you're making.

File: src/RemoteStorageService/Confirmation.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    expect(getByText('ui-inventory.remote.confirmation.heading')).toBeInTheDocument();
+  });
+  it('should render remote items message', () => {
+    render(
+      <IntlProvider>
+        <Message />
+      </IntlProvider>
+    );
+    expect(screen.getByText('ui-inventory.remote.confirmation.message')).toBeInTheDocument();
+  });
```
Comment (zburke) [inline src/RemoteStorageService/Confirmation.test.js:22]: Thinking through this a little more, there should really be two tests here: 

1. without a `count` prop, Message renders the expected default value
2. with a `count` prop, Message renders the given value

**Comment by zburke** [comment]
@KetineniM, you should have permission to push your branches directly to the `folio-org/ui-inventory` remote instead of working from a fork. Please let me know if this is not working for you.

File: src/RemoteStorageService/Confirmation.test.js (hunk 1 above)
Comment (KetineniM) [reply src/RemoteStorageService/Confirmation.test.js]: Zak,
We made changes as you suggested
Please review and let me know if any.

**Review by zburke — CHANGES_REQUESTED** [review]
Hmmm, this mock didn't work out the way I expected. As implemented, it's hard-coded to this specific test which is not a good option. 

Let's put this component aside for now and move on to others. I need to think about the best option here.

**Comment by mkuklis** [comment]
@KetineniM and @zburke I will try to help out with these PRs so we can move forward and merge them.

@KetineniM as Zak mentioned could you please start by opening a new PR with the same work directly under https://github.com/folio-org (you can just clone it directly from https://github.com/folio-org/ui-inventory.git) instead of working from your personal fork?

This will help us streamline this process a bit. 

After you reopen the PR I will start reviewing it.

**Comment by KetineniM** [comment]
@mkuklis - Hi, thank you for your support, I tried to start new PR, getting the below error, please see the screen shot

image

**Comment by mkuklis** [comment]
Hi, @KetineniM I'm double-checking this with the devops team. My understanding was that you already have access.

**Comment by mkuklis** [comment]
@KetineniM please try again. It should be all set.

**Pull request closed without merging**